libs/application/templates/public-debt-payment-plan/src/fields/Overview/Overview.tsx

```diff
@@ -0,0 +1,90 @@
+import { FieldBaseProps } from '@island.is/application/core'
+import { Label, ReviewGroup } from '@island.is/application/ui-components'
+import { Box, GridColumn, GridRow, Text } from '@island.is/island-ui/core'
+import React from 'react'
+import { PaymentPlanExternalData } from '../../lib/dataSchema'
+import { PaymentPlanTable } from '../components/PaymentPlanTable/PaymentPlanTable'
+import { useMockPaymentPlan } from '../PaymentPlan/useMockPaymentPlan'
+
+export const Overview = ({ application }: FieldBaseProps) => {
+  const externalData = application.externalData as PaymentPlanExternalData
+  const paymentPlanList = (application.externalData as PaymentPlanExternalData)
+    .paymentPlanList
+
+  const editAction = () => {
+    // TODO: Write better function. What will happen on edit?
+    console.log('this is edit action')
+  }
+
+  // TODO: Add text to messages.
+  // TODO: Add edit functionality.
+  return (
+    <>
+      <ReviewGroup isEditable editAction={editAction}>
+        <GridRow>
+          <GridColumn span={['6/12', '5/12']}>
+            <Box>
+              <Label>Nafn</Label>
+              <Text>Fjármundur Skuldason</Text>
```

Comment:
Shouldn't this be provided by the application.answers ?

Reply:
Yup, that's why it's in the TODO above. This is not completely finished.